pages.fr/common/grep.md

```diff
@@ -0,0 +1,36 @@
+# grep
+
+> Recherche des motifs dans un texte.
+> Supportes des motifs simple et des expressions rationnelles.
+
+- Recherche une chaîne de caractères précise :
+
+`grep {{search_string}} {{chemin/vers/fichier}}`
```

Comment:
```suggestion
`grep {{chaîne_recherchée}} {{chemin/vers/fichier}}`
```
This should also be translated. Does "chaîne recherchée" make sense? Or perhaps "chaîne à rechercher"?

NB: commenting only once but it applies to all the occurrences in the page.